docs/_documentations/troubleshooting.md

```diff
@@ -1002,6 +1002,20 @@ If you can run `docker` from the command line, but Codewind still fails to find
 3. Open the terminal you ran `docker` from and run `code`.
 4. Now, the new VS Code instance shares the terminal's `PATH`. Start Codewind again.
 
+## VS Code extension fails to start
+<!--
+Action/Topic: VS Code extension fails to start
+Issue type: bug
+Issue link: https://github.com/eclipse/codewind/issues/2738
+Solution: https://github.com/eclipse/codewind/issues/2738#issuecomment-618909203
+Info added in 0.13.0.
+-->
+You install VS Code, it fails to start, and reports a `Version in "/home/ibmuser/.codewind/docker-compose.yaml" is unsupported` error message. The main symptom of this issue is that the `~/codewind/docker-compose.yaml` file is not created when you are installing Codewind. 
+
+**Workaround:**
+1. To fix your Ubuntu VM, update the `docker-compose` version to the latest, and then uninstall and reinstall Codewind.
+2. For macOS, remove the Codewind container that shows when you enter the `$ docker ps -all` command, and then uninstall and reinstall Codewind. 
```

Comment:
Ah, I'm now thinking both of these workarounds are not specific to the OS, they're two steps that could be checked on any OS.

> 1. Check your `docker-compose` version is the latest, and then uninstall and reinstall Codewind.
> 2. If there is a Codewind container showing in the output of `$ docker ps -all` , remove the container. Then uninstall and reinstall Codewind. 

What do you think? Other than that this looks good, thank you

Reply:
Yes, this is better, have made the changes. Thanks.